.github/workflows/wheel_macos_arm64.yml

```diff
@@ -8,36 +8,18 @@ on:
     types: [published]
 
 env:
-  CIBW_BUILD: 'cp37-* cp38-* cp39-* cp310-*'
-
   ARCHS: 'arm64'
 
-  # MacOS specific build settings
-  CIBW_BEFORE_ALL_MACOS: |
-    brew uninstall --force oclint
-
-  # Python build settings
-  CIBW_BEFORE_BUILD: |
-    pip install pybind11 ninja cmake
-
-  # Testing of built wheels
-  CIBW_TEST_REQUIRES: numpy~=1.21 scipy pytest pytest-cov pytest-mock flaky
-
-  CIBW_BEFORE_TEST: pip install git+https://github.com/PennyLaneAI/pennylane.git@master
-
-  CIBW_TEST_COMMAND: |
-    pl-device-test --device=lightning.qubit --skip-ops -x --tb=short --no-flaky-report
-
-  CIBW_BUILD_VERBOSITY: 1
-
 jobs:
   mac-wheels-arm64:
     strategy:
       fail-fast: false
       matrix:
         os: [macos-11]
         arch: [arm64]
-    name: macos-latest::arm64
+        cibw_build: [cp38-*, cp39-*, cp310-*]
+
+    name: macos-latest::arm64 (Python ${{ fromJson('{ "cp37-*":"3.7","cp38-*":"3.8","cp39-*":"3.9","cp310-*":"3.10" }')[matrix.cibw_build] }})
```

Comment:
We should probably double-check the wheels from this on @isschoch or @multiphaseCFD Macs before merging.

Reply:
I will wait for their feedback.

Reply:
Hi @multiphaseCFD, could you please present a log of the tests so we can analyze the failed testing cases?